recipes/neuraxle/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "neuraxle" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8dc696bf67f9b45be778eb2703221525aed94534da704a79ced3e531ffef5fce
```

Comment:
Probably need to update the checksum too

Reply:
Fixed! Note to self: back to the documentation if needed again for next version: 
https://conda-forge.org/docs/maintainer/adding_pkgs.html#step-by-step-instructions

Reply:
Re: had to do the following curl to get it working: 
- `curl -sL https://pypi.io/packages/source/n/neuraxle/neuraxle-0.8.1.tar.gz | openssl sha256`